opentreemap/stormwater/templates/stormwater/RainBarrel_detail.html

```diff
@@ -0,0 +1,9 @@
+{% extends "treemap/resource_detail.html" %}
+{% load i18n %}
+{% load l10n %}
+{% load form_extras %}
+
+{% block resource_details %}
+{% trans "Capacity in gallons" as capacity %}
```

Comment:
Similar to Rick's comments above, it would be ideal to avoid mentioning the units in the hardcoded label, since it can be changed to liters.